stubs/networkx/METADATA.toml

```diff
@@ -0,0 +1,4 @@
+version = "3.1"
+upstream_repository = "https://github.com/networkx/networkx"
+requires = ["pandas", "numpy", "scipy", "pygraphviz"]
+partial_stub = true
```

Comment:
Unless there's an issue with stubgen not being able to generate all the stubs, this would probably not be a partial stub. (see https://peps.python.org/pep-0561/#partial-stub-packages for more info on what are partial stub packages)

I also see a ton of usage of `Any`. It is usually used when it's not possible to accurately type an item using the current type system. It should be used sparingly and we try to avoid it. `Incomplete` is a placeholder where a proper type might be substituted. It's a "to do" item and should be replaced if possible. See  [Incomplete annotations](Incomplete is a placeholder where a proper type might be substituted. It's a "to do" item and should be replaced if possible.)

I'd recommend running the `python scripts/create_baseline_stubs.py networkx` script, which takes care of a handful of setup for new stubs, and seeing what's different from your existing stubs. See [Auto-generating stub files](https://github.com/python/typeshed/blob/main/CONTRIBUTING.md#auto-generating-stub-files) from the contributing doc.
For instance, it'll automatically use the `Incomplete` alias when needed, and otherwise leave the type annotation empty when unknown. Add default param values when found in the source. Add the new stub to the non-strict pyright configuration, etc.